code/modules/antagonists/pirate/pirate.dm

```diff
@@ -37,6 +37,18 @@
 		objectives |= crew.objectives
 	. = ..()
 
+/datum/antagonist/pirate/apply_innate_effects(mob/living/mob_override)
+	. = ..()
+	var/mob/living/owner_mob = mob_override || owner.current
+	var/datum/language_holder/holder = owner_mob.get_language_holder()
+	holder.grant_language(/datum/language/piratespeak, TRUE, TRUE, LANGUAGE_PIRATE)
+	holder.selected_language = /datum/language/piratespeak
+
+/datum/antagonist/pirate/remove_innate_effects(mob/living/mob_override)
+	. = ..()
```

Comment:
This should be called last as `return ..()` since you are removing the innate effects